cli/src/main/java/com/box/l10n/mojito/cli/command/L10nJCommander.java

```diff
@@ -124,21 +147,30 @@ public void run(String... args) {
       try {
         command.run();
       } catch (SessionAuthenticationException ae) {
+        String message = "Invalid username or password";
         logger.debug("Exit with Invalid username or password", ae);
-        printErrorMessage("Invalid username or password");
+        printErrorMessage(message);
+        this.notifyFailure(command, args, message);
         exitWithError();
       } catch (CommandWithExitStatusException cwese) {
-        logger.error("Exit with error for command: " + command.getName(), cwese);
+        String message = "Exit with error for command: " + command.getName();
+        logger.error(message, cwese);
+        this.notifyFailure(command, args, cwese);
         exitWithError(cwese.getExitCode());
       } catch (CommandException ce) {
+        String message = "Exit with error for command: " + command.getName();
         printErrorMessage(ce.getMessage());
-        logger.error("Exit with error for command: " + command.getName(), ce);
+        logger.error(message, ce);
+        if (!ce.isExpected()) {
```

Comment:
`isExpected` should be changed as mentioned earlier

Reply:
Done